django/contrib/postgres/search.py

```diff
@@ -300,3 +300,18 @@ class TrigramSimilarity(TrigramBase):
 class TrigramDistance(TrigramBase):
     function = ''
     arg_joiner = ' <-> '
+
+
+class TrigramWordSimilarity(Func):
+    function = 'WORD_SIMILARITY'
+    output_field = FloatField()
+
+    def __init__(self, expression, string, **extra):
+        if not hasattr(string, 'resolve_expression'):
+            string = Value(string)
+        super().__init__(string, expression, **extra)
+
+
+class TrigramWordDistance(TrigramBase):
+    function = ''
+    arg_joiner = ' <->> '
```

Comment:
:thinking: 
```suggestion
    arg_joiner = ' <<-> '
```
Paolo, Can you take a look? (\cc @pauloxnet) :point_up:

Reply:
This is for the same reason as the argument swap in `TrigramWordSimilarity`

Reply:
I wonder if using the `<->>` operator would use any indexes 🤔

Reply:
I just made a quick test on PostgreSQL 12.5 and it seems the query planner is smart enough to use the index with both operators. So i think we should use the order that make the most sense (i would suggest have the same order as `WORD_SIMILARITY`)

<details>
<summary>Details of both queries and EXPLAIN</summary>

```sql
EXPLAIN ANALYSE
SELECT id, search <->> 'findme' as dist
FROM public.catalog_index
ORDER BY dist
LIMIT 10;
```


```sql
EXPLAIN ANALYSE
SELECT id, 'findme' <<-> search as dist
FROM public.catalog_index
ORDER BY dist
LIMIT 10;
```

Both queries give me the exact same plan:
```
Limit  (cost=0.28..9.69 rows=10 width=20) (actual time=5.884..7.119 rows=10 loops=1)
  ->  Index Scan using catalog_index_search_gist_idx on catalog_index  (cost=0.28..11300.31 rows=12002 width=20) (actual time=5.883..7.115 rows=10 loops=1)
        Order By: (search <->> 'findme'::text)
Planning Time: 0.105 ms
Execution Time: 7.147 ms
```

</details>